server/notification-providers/hostedsms.js

```diff
@@ -0,0 +1,48 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+const { UP, DOWN } = require("../../src/util");
+
+class HostedSMS extends NotificationProvider {
+
+    name = "hostedsms";
+
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+        let okMsg = "Sent Successfully.";
+
+        try {
+            let config = {
+                headers: {
+                    "Content-Type": "application/json",
+                }
+            };
+
+            let textMsg = "";
+            if (heartbeatJSON && heartbeatJSON.status === UP) {
+                textMsg = `✅ [${monitorJSON.name}] is back online`;
+            } else {
+                textMsg = `🔴 [${monitorJSON.name}] went down`;
+            }
```

Comment:
Please add a short comment before this statement summarizing the contents of https://github.com/louislam/uptime-kuma/pull/3176#discussion_r1199601497

Reply:
Maybe this will be good.